static/styles/products.css

```diff
@@ -0,0 +1,142 @@
+body{
+    font-family: Dosis;
+    box-sizing: border-box;
+}
+.product{
+    margin:5%;
+    border-bottom-style: dashed;
+}
+.products_description{
+    display: flex;
+}
+.products_description>section{
+    width:50%
+}
+.products_description>section>img{
+    width: 100%;
+    object-fit: cover;
+}
+button{
+    width:100%;
```

Comment:
button should look same across all webpages. So no need to add multiple same css rule on each page. Create a common styles file and import it wherever necessary